### ISSUE TITLE
predict function

### ISSUE THREAD

**Issue by gianmarco-terrones** [hspark1212/MOFTransformer#134]
Sorry for all the raised issues, I appreciate your help in getting these questions answered. When I run the predict function as described [here](https://hspark1212.github.io/MOFTransformer/getting_started/predict.html), I get a test_prediction.csv file that looks like this:
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>

<meta http-equiv="Content-Style-Type" content="text/css">
</head>
<body>

cif_id | regression_logits | regression_labels
-- | -- | --
ACOFUU | -0.6171875 | 1
ADABAK | -0.0834961 | 2
AFEJUQ | -0.3359375 | 1
ELUBEU | 0.28710938 | 0

</body>
</html>

 
However, I trained the model with the keyword arguments `loss_names="classification"` and `n_classes=3` in the run function. When I use these settings in the predict function, the test_prediction.csv looks like this:
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/gianmarcoterrones/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/gianmarcoterrones/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">

</head>

<body link="#0563C1" vlink="#954F72">

classification_logits | classification_labels
-- | --
[-2.6875, 3.828125, -0.734375] | 1
[-2.71875, 4.0, -1.4375] | 2
[-2.59375, 1.34375, 2.0625] | 1
[-0.291015625, -2.46875, 2.8125] | 0

</body>

</html>

I have a few questions about this. Could the CSV file for classification predictions be modified to include the cif_id? Also, how should I interpret the prediction? In the list in the classification_logits column, is the index of the largest entry the prediction (e.g. for [-2.6875, 3.828125, -0.734375] is the prediction class 1)?

Also, a minor point: the docstring of the predict function has `load_path` listed twice, and I believe its second appearance as a keyword argument should be removed.

**Comment by Yeonghun1675** [comment]
Hi @gianmarco-terrones,

Thank you for your interest in our MOFTransformer, we're always looking to improve our MOFTransformer based on your feedback.

For the corresponding classification logits, we have labeled the last tensor that can be obtained from the classification head, where the index with the highest value is the model's prediction.

For convenience, the new version adds classification_logits_index, which indicates which class the model predicted. We've also modified classification_logits to return a softmaxed value to more intuitively show what weight the model predicted.
